cli/npm/managed/resolvers/local/bin_entries.rs

```diff
@@ -68,6 +262,17 @@ fn symlink_bin_entry(
   let link = bin_node_modules_dir_path.join(bin_name);
   let original = package_path.join(bin_script);
 
+  if !original.exists() {
```

Comment:
These checks take extra time. Is it worth doing? Maybe we should just set it up and if it errors then it errors?

By the way, I think maybe removing the next if statement and just setting it up each time might be faster than doing a read.

Reply:
Makes sense, I moved these into the error handling instead of checking for ahead of time.